mm2src/coins/lightning.rs

```diff
@@ -919,6 +957,39 @@ pub async fn open_channel(ctx: MmArc, req: OpenChannelRequest) -> OpenChannelRes
     })
 }
 
+#[derive(Deserialize)]
+pub struct UpdateChannelReq {
+    pub coin: String,
+    pub channel_id: H256Json,
+    pub counterparty_node_id: PublicKeyForRPC,
+    pub options: ChannelOptions,
+}
+
+/// Updates configuration for an open channel.
+pub async fn update_channel(ctx: MmArc, req: UpdateChannelReq) -> UpdateChannelResult<String> {
```

Comment:
What about providing proper result type instead of plain text? Or maybe even `bool` type can be nice.

Reply:
Done